reloj-checador/app/views/attendaces/_form.html.erb

```diff
@@ -17,11 +17,9 @@
 </div>
 
 <script>
-  let clock = document.getElementById("clock")
-  
+  var clock = document.getElementById("att-new-clock");
   setInterval(() => {
     let date = new Date()
     clock.innerHTML = date.toLocaleTimeString();
-    console.log(date.toLocaleTimeString());
   },1000);
 </script>
```

Comment:
One suggestion to keep your code clean is to distribute the elements. In this case, the javascript code in a JS file.